rakelib/test.rake

```diff
@@ -87,7 +87,7 @@ namespace "test" do
     require "simplecov"
     SimpleCov.start do
       # Skip non core related directories and files.
-      ["vendor/", "spec/", "bootstrap/rspec", "Gemfile", "gemspec"].each do |pattern|
+      ["vendor/", "test/", "bootstrap/rspec", "Gemfile", "gemspec"].each do |pattern|
```

Comment:
I changed it because some folks complained to me when I first introduced this idea that specs was misunderstood, but happy to see this also confuses you, so I qill bring back the spec dir as I understand is much clear to "ruby" folks.